lib/internal/errors.js

```diff
@@ -1182,6 +1182,21 @@ E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s', TypeError);
 E('ERR_UNKNOWN_FILE_EXTENSION', 'Unknown file extension: %s', TypeError);
 E('ERR_UNKNOWN_MODULE_FORMAT', 'Unknown module format: %s', RangeError);
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s', TypeError);
+E('ESM_CJS_LOAD_ERROR', (error) => {
+
+  if(error instanceof SyntaxError && (/^import/mi).test(error.stack)){
+    const sp =  error.stack.split('\n');
+    const filepath = sp[0];
+    const line = `${sp[1]}\n${sp[2]}`
+    return `${filepath}
+${line}
+Warning: Attempted to load commonjs module but found static import statement
```

Comment:
The important thing with the error message is to provide the user the **immediate** fix to get their code to run.

We are all lazy developers, we don't want to read docs, we just want to get our code running :)

So perhaps something like - 

```suggestion
Note: To load an ES module set "type": "module" in the package.json or use the .mjs extension.
```